demisto_sdk/commands/common/errors.py

```diff
@@ -2365,3 +2366,10 @@ def wrong_link_in_wizard(link):
     @error_code_decorator
     def wizard_integrations_without_playbooks(integrations: set):
         return f'The following integrations are missing a set_playbook: {integrations}'
+
+    @staticmethod
+    @error_code_decorator
+    def pack_should_be_hidden(pack_name: str, deprecated_content_items: dict):
+        deprecated_items = ' and '.join([f'{key}: {val}' for key, val in deprecated_content_items.items()])
+        return f'Pack {pack_name} should be hidden, the following {deprecated_items} are deprecated.\n' \
+               f'Pack must be hidden if all of its integrations, playbooks and scripts are deprecated.'
```

Comment:
```suggestion
        return f'Pack {pack_name} should be hidden, as all its integrations, playbooks and scripts are deprecated.'
```

then there's no need for `deprecated_content_items`

Reply:
I guess you are right, i thought it could be good to show the edge user the entire content entitles which are deprecated, however you are right, that's redundant